go/kbnm/host_json.firefox.template

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "io.keybase.kbnm",
+  "description": "Keybase Native Messaging API",
+  "path": "@@HOST_PATH@@",
+  "type": "stdio",
+  "allowed_extensions": [
+    "keybase@keybase.io"
```

Comment:
Going through the Firefox submission flow next, wanted the installer done so I could actually test the installation.